src/ui/component/sideBar/view.jsx

```diff
@@ -68,7 +76,7 @@ function SideBar(props: Props) {
               ),
             },
           ].map(linkProps => (
-            <li key={linkProps.label}>
+            <li key={buildKey(linkProps.label)}>
```

Comment:
We should avoid using `random` values for keys in React components. Each time it's rendered it will have a different key, which will cause issues if we ever add options to change the order of the menu.

I think we can just change this to `linkProps.navigate`?

Reply:
That's a great point. I'll make this change